packages/components-react/tsconfig.base.json

```diff
@@ -11,7 +11,7 @@
     "jsx": "react-jsx",
     "lib": ["DOM", "ES2022"],
     "module": "ESNext",
-    "moduleResolution": "Bundler",
+    "moduleResolution": "NodeNext",
```

Comment:
`"module": "ESNext"` + `"moduleResolution": "NodeNext"` is not a valid combination. Did you have an issue that was solved by changing this?

Reply:
Yes, I couldn't run `pnpm run build` nor `pnpm run test`

Reply:
What error did you get? `Bundler` was added in TS5 so if your installed versions of TS were stuck on 4, that could explain things.

Reply:
I changed it back. I probably forgot to `pnpm i -f` beforehand. Running into different issues now, though. The `@utrecht/calendar-react` package cannot be found. Not even when I explicitly (manually) build it. It is in the dependencies, so it shouldn't be an issue. 

Just keep digging